.phan/config.php

```diff
@@ -38,9 +38,15 @@
         "PhanUnusedPublicNoOverrideMethodParameter",
         "PhanTypePossiblyInvalidDimOffset",
         "PhanUndeclaredMethod",
+        "PhanTypeMismatchDeclaredParam",
```

Comment:
Why are these re-ignored? A lot of work went into fixing each of the phan rules that aren't ignored and Travis should have prevented any regressions from happening.